vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -47,6 +47,13 @@ class Graph(VizroBaseModel):
     _set_actions = _action_validator_factory("clickData")
     _validate_callable = validator("figure", allow_reuse=True)(_process_callable_data_frame)
 
+    @validator("figure")
```

Comment:
I can't remember anymore if it runs all validators even if one doesn't pass. If it doesn't do that, I would move this validator above the `_validate_callable`, such that this one is not even run anymore? However, if it runs all validators regardless if one passes, then it doesn't matter